src/App.php

```diff
@@ -2,33 +2,50 @@
 
 namespace TrendingRepos;
 
-class App {
-
-    /**
-     * @var array
-     */
-    protected static $registry = [];
+use TrendingRepos\Exception\RouterException;
+use TrendingRepos\Core\Request;
+use TrendingRepos\Core\Router;
+
+class App 
+{
+    protected $environment = 'development';
+    public $registry = [];
+    protected $router;
```

Comment:
The same recommendation as for methods ordering applies to the properties as well. Declare public first, then protected ones and finally private ones. See [Symfony coding standards](https://symfony.com/doc/current/contributing/code/standards.html#structure) (again).